process_pr.py

```diff
@@ -630,6 +630,7 @@ def check_matrix_extras(first_line, repo, params, mkey, param, *args):
     print(first_line, repo, params, mkey, param)
     if kitem[-1] in EXTRA_RELVALS_TESTS:
         param = param + "_" + kitem[-1].upper().replace("-", "_")
+    first_line = ",".join(sorted(first_line.split(",")))
```

Comment:
@iarspider , should we drop trim (removing space char) each item too?

Reply:
We can, or we can wait until next PR, in which we will normalize whitespaces, remove `@cmsbuild please` prefix and remove spaces around commas.